packages/block-library/src/pattern/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import PatternEdit from './edit';
+
+const { name } = metadata;
+export { metadata, name };
+
+export const settings = {
+	edit: PatternEdit,
+	save: () => {
+		return <InnerBlocks.Content />;
+	},
```

Comment:
Thanks to the logic in PatternEdit to replace the block with its contents, I think we can remove `save` entirely.

As far as I can tell, there should never be a case where there are inner blocks to serialise for this block. I even think that allowing Pattern to serialise any kind of content here opens the door for bugs where the `edit` side wants to use the freshest pattern from `__experimentalGetParsedPattern` while `save` has saved something else to the database.

What do you think?

Reply:
Yeah that makes sense.